extra/write_throughput_benchmark.rst

```diff
@@ -0,0 +1,135 @@
+:orphan:
+
+.. _citus_write_throughput_benchmark:
+
+Benchmark Setup with Citus and pgbench
+--------------------------------------
+
+In this section, we provide step by step instructions to benchmark Citus' write throughput. For these benchmark steps, we use `Citus Cloud <https://console.citusdata.com/users/sign_up>`_ to create test clusters, and a standard benchmarking tool called `pgbench  <https://www.postgresql.org/docs/current/static/pgbench.html>`_.
+
+If you're interested in general throughput numbers based on these tests, you can also find them in :ref:`scaling_data_ingestion`.
+
+Create Citus Cluster
+~~~~~~~~~~~~~~~~~~~~
+
+The easiest way to start a Citus Cluster is by vising the Citus Cloud dashboard. This dashboard allows you to choose different coordinator and worker node configurations and charges you by the hour. Once you picked your desired cluster setup, click on the "Create New Formation" button.
+
+.. image:: ../images/cloud-coordinator-worker-slider.png
+  :scale: 50 %
+  :align: center
+
+A pop-up will ask you the AWS region (US East, US West) where your formation will be created. Please remember the region where you created your Citus Cloud formation. We will use it in the next step.
+
+Please note that Citus Cloud automatically tunes your cluster based on your hardware configuration. If you're planning to run the following steps on your own cluster, you will need to manully increase :code:`max_connections = 303` on the coordinator and worker nodes.
+
+Create an Instance to Run pgbench
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+pgbench is standard benchmarking tool provided by PostgreSQL. pgbench runs given SQL commands repeatedly and measures number of completed transactions per second.
+
+Since pgbench itself uses some CPU power to execute SQL commands, we recommend that you run pgbench on a separate machine than the one where your Citus cluster runs. This recommendation also follows pgbench's documentation.
```

Comment:
Since pgbench itself consumes CPU power, we recommend running it on a separate machine from those running your Citus cluster. As pgbench's documentation points out, putting them on the same machine can skew test results.